Dockerfile.debug

```diff
@@ -9,6 +9,7 @@ WORKDIR /go/src/metacontroller/
 ENV ADDITIONAL_BUILD_ARGUMENTS='-gcflags="all=-N -l"'
 RUN make install
 RUN go get github.com/go-delve/delve/cmd/dlv
+RUN go install github.com/go-delve/delve/cmd/dlv
```

Comment:
hmm I think line above is doing the same...

Reply:
@grzesuav It looks like the functionality changed in go 1.18: https://tip.golang.org/doc/go1.18#modules
> go get
go get no longer builds or installs packages in module-aware mode. go get is now dedicated to adjusting dependencies in go.mod. Effectively, the -d flag is always enabled. To install the latest version of an executable outside the context of the current module, use [go install example.com/cmd@latest](https://golang.org/ref/mod#go-install). Any [version query](https://golang.org/ref/mod#version-queries) may be used instead of latest. This form of go install was added in Go 1.16, so projects supporting older versions may need to provide install instructions for both go install and go get. go get now reports an error when used outside a module, since there is no go.mod file to update. In GOPATH mode (with GO111MODULE=off), go get still builds and installs packages, as before.

I was confused by this as well 😂 . I updated it just use the install command now.